packages/core/upload/admin/src/utils/tests/typeFromMime.test.ts

```diff
@@ -14,6 +14,30 @@ describe('typeFromMime', () => {
     expect(type).toBe(AssetType.Video);
   });
 
+  it('gives a type of pdf when the mime contains "pdf"', () => {
+    const type = typeFromMime('application/pdf');
+
+    expect(type).toBe(DocType.Pdf);
+  });
+
+  it('gives a type of csv when the mime contains "csv"', () => {
+    const type = typeFromMime('text/csv');
+
+    expect(type).toBe(DocType.Csv);
+  });
+
+  it('gives a type of xls when the mime contains "excel"', () => {
+    const type = typeFromMime('application/vnd.ms-excel');
+
+    expect(type).toBe(DocType.Xls);
+  });
+
+  it('gives a type of zip when the mime contains "zip"', () => {
+    const type = typeFromMime('application/zip');
+
+    expect(type).toBe(DocType.Zip);
+  });
+
   it('gives a type of document when the mime is neither video nor image', () => {
```

Comment:
this test is failing right now
`typeFromMime › gives a type of document when the mime is neither video nor image

    expect(received).toBe(expected) // Object.is equality

    Expected: "doc"
    Received: "pdf"`